package.json

```diff
@@ -5,11 +5,15 @@
         "react": "^15.6.1",
         "react-dom": "^15.6.1"
     },
-    "scripts": {},
+    "scripts": {
+        "build": "webpack",
+        "watch": "webpack --watch"
```

Comment:
If somebody (e.g. PHP developer) doesn't have webpack installed globally it can use `yarn build` instead of typing `node ./node_modules/.bin/webpack`. It also, for me, works as some kind of documentation where you can check how to build the project. Now it is obvious but e.g in the future command may change etc. so I think it is good practice to put these scripts in `package.json`.